app/assets/javascripts/details/shell/c2.js

```diff
@@ -166,6 +166,12 @@ C2 = (function() {
     }
   }
 
+  C2.prototype.createErrorNotifications = function(errors, title){
```

Comment:
Instead of making this, can you use https://github.com/18F/C2/blob/74adab79d998e36b442c3b5bfe3df0e2891d3cc6/app/assets/javascripts/details/shell/c2.js#L161-L167